Makefile

```diff
@@ -4,6 +4,7 @@ RESOURCES := $(shell find resources)
 target/server-%.jar: $(SOURCES) $(RESOURCES) prepare
 	clojure -T:build uber
 
+.PHONY: uberjar
```

Comment:
What does .PHONY signify in a Makefile?

Reply:
That the target is not a filename that the rule will create (which is the default). Without this, if a file named `uberjar` were created and had a newer timestamp than the prerequisites, make would say "nothing to do for uberjar" when you ran `make uberjar`. Make calls these "phony targets."

Reply:
Thanks, that makes sense.